<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="EPBYMINW2706" ignored="0" name="Multiplication Cal Test" tests="8" failures="1" timestamp="28 May 2019 13:32:23 GMT" time="0.401" errors="0">
  <testcase name="multiplicationTest" time="0.0" classname="com.epam.cdp.cal.tests.MultiplicationCalculatorTest"/>
  <testcase name="multiplicationTest" time="0.0" classname="com.epam.cdp.cal.tests.MultiplicationCalculatorTest"/>
  <testcase name="multiplicationTest" time="0.0" classname="com.epam.cdp.cal.tests.MultiplicationCalculatorTest"/>
  <testcase name="multiplicationTest" time="0.0" classname="com.epam.cdp.cal.tests.MultiplicationCalculatorTest"/>
  <testcase name="multiplicationTest" time="0.0" classname="com.epam.cdp.cal.tests.MultiplicationCalculatorTest">
    <failure type="java.lang.AssertionError" message="Invalid result of multiplication expected [15.05] but found [15.0]">
      <![CDATA[java.lang.AssertionError: Invalid result of multiplication expected [15.05] but found [15.0]
at com.epam.cdp.cal.tests.MultiplicationCalculatorTest.multiplicationTest(MultiplicationCalculatorTest.java:12)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
... Removed 14 stack frames]]>
    </failure>
  </testcase> <!-- multiplicationTest -->
  <testcase name="multiplicationTest" time="0.0" classname="com.epam.cdp.cal.tests.MultiplicationCalculatorTest"/>
  <testcase name="multiplicationTest" time="0.0" classname="com.epam.cdp.cal.tests.MultiplicationCalculatorTest"/>
  <testcase name="multiplicationTest" time="0.0" classname="com.epam.cdp.cal.tests.MultiplicationCalculatorTest"/>
</testsuite> <!-- Multiplication Cal Test -->
